tests/test_run_project.py

```diff
@@ -0,0 +1,12 @@
+import subprocess
+
+
+def test_pytask_python(cookies, basic_project_dict):
+
+    result = cookies.bake(extra_context=basic_project_dict)
+    subprocess.run("conda develop . && pytask", shell=True, cwd=result.project)
```

Comment:
@tobiasraabe -- is this how you would test whether pytask works correctly or is there a smarter way? (pls ignore the rest of the test, does not make sense yet)

Reply:
Seems fine. I think ``shell = True`` is in general discouraged, but for security which is not important here anyway (https://stackoverflow.com/a/3172488).